### PR TITLE
PEM - install clarification

### PR DESCRIPTION
**Pull request by nidhibhammar**

### CODE REVIEW

File: product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx [hunk 1]
```diff
@@ -5,56 +5,71 @@ navTitle: "Prerequisites"
...
 Postres Enterprise Manager requires a Postgres server running either [EDB Postgres Advanced Server](/epas/latest) or [PostgresSQL](/supported-open-source/postgresql/) version 11 or later to hold the Postgres Enterprise Manager backend database. 
   
-  1. Install the Postgres server. See [Installing EDB Postgres Advanced Server on Linux](/epas/latest/epas_inst_linux) or [Installing PostgreSQL](/supported-open-source/postgresql/installer/).
+  1. Set up the repository
+
+     Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
+
+     -  To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+
+     -  To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
```
Comment (sonotley) [inline product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx:14]: We could add a note here that this is only required if you are not using an EPAS backend (and even then you can sometimes do without it as Jun found!)

File: product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx [hunk 2]
```diff
@@ -5,56 +5,71 @@ navTitle: "Prerequisites"
...
+          zypper install sslutils_<x> postgresql<x>-contrib
           ```
 
-          Where `x` is the Postgres server version.
+          Where `<x>` is the Postgres server version.
 
-      !!! Note
-          Debian 10 and Ubuntu 20 increased the requirements to accept the certificates due to security reason.
+      !!! Note For Debian and Ubuntu users     
+          The postgres community repository for apt (used by Debian and Ubuntu) does not include `sslutils` and therefore this must be installed from the EDB repository along with [PEM](/installing_pem_server_using_edb_repository/index.mdx).
```
Comment (sonotley) [inline product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx:68]: This probably isn't needed anymore as the repos are both installed.

File: product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx [hunk 3]
```diff
@@ -5,56 +5,71 @@ navTitle: "Prerequisites"
...
+          
+          Debian 10 and Ubuntu 20 changed the requirements for accepting certificates.
 
-            -   If you want to install the PEM agent on any of the machines, you must upgrade ssltuils to 1.3 where 4096-bit RSA key and sha256 signature algorithm support was added.
+            -   If you want to install the PEM agent on any of the machines, you must upgrade sslutils to 1.3 where 4096-bit RSA key and sha256 signature algorithm support was added.
```
Comment (sonotley) [inline product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx:72]: I know this isn't new in this PR, but I find the talk of 'upgrading' confusing given that these instructions are for a fresh install and that all the repos (I think) give v1.3 by default. Do we need to mention this at all? If so maybe we can clarify that it only applies if you already have an old version of SSLUtils?

**Review by sonotley — COMMENTED** [review]
Looks good overall, left some comments for consideration.

File: product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx [hunk 4]
```diff
@@ -5,56 +5,71 @@ navTitle: "Prerequisites"
...
+
+
+  2. Install the Postgres server. See [Installing EDB Postgres Advanced Server on Linux](/epas/latest/epas_inst_linux) or [Installing PostgreSQL](/supported-open-source/postgresql/installer/).
   
-  2. Configure authentication on the Postgres server by updating the `pg_hba.conf` file. The `pg_hba.conf` file manages connections. Verify that the `pg_hba.conf` file allows connections from the PEM server, the monitoring PEM agent, and the host of the Apache web server server.
+  3. Configure authentication on the Postgres server by updating the `pg_hba.conf` file. The `pg_hba.conf` file manages connections. Verify that the `pg_hba.conf` file allows connections from the PEM server, the monitoring PEM agent, and the host of the Apache web server server. Pointers for configuring access:
   
+     - PEM server connects to the PEM backend database as a superuser using password authentication. On Linux platforms, this requires you to add a new superuser that authenticates using a password.
+     
+     - To allow the new superuser to connect using password authentication, add a line `pg_hba.conf` that allows `host` connections using `md5` authentication.
```
Comment (sonotley) [inline product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx:23]: I should have said md5 *or scram-sha-256* authentication. The default changed in pg14 I think.

File: product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx (hunk 2 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx:68]: I think if they have installed Postgres using Postgres community repos then this note is required.

File: product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx (hunk 3 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx]: updated and committed

File: product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx (hunk 4 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx]: updated and committed

File: product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx (hunk 1 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx:14]: added note and committed

File: product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx (hunk 2 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx:70]: @sonotley, If you agree with this, we can take this PR further if not, let's discuss this over call.

Comment (sonotley) [reply product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx:70]: agreed

File: product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx [hunk 5]
```diff
@@ -5,56 +5,73 @@ navTitle: "Prerequisites"
...
 
-      !!! Note
-          Debian 10 and Ubuntu 20 increased the requirements to accept the certificates due to security reason.
+      !!! Note For Debian and Ubuntu users     
+          The postgres community repository for apt (used by Debian and Ubuntu) does not include `sslutils` and therefore this must be installed from the EDB repository along with [PEM](/installing_pem_server_using_edb_repository/index.mdx).
+          
+          Debian 10 and Ubuntu 20 changed the requirements for accepting certificates.
 
-            -   If you want to install the PEM agent on any of the machines, you must upgrade ssltuils to 1.3 where 4096-bit RSA key and sha256 signature algorithm support was added.
+            -   If you want to install the PEM agent on any of the machines with an old version of sslutils, then you must upgrade sslutils to 1.3. Version 1.3 has a 4096-bit RSA key and sha256 signature algorithm support added to it.
```
Comment (sonotley) [inline product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx:74]:
```suggestion
            -   If you want to install the PEM agent on a machine with an old version of sslutils, then you must upgrade sslutils to 1.3. Version 1.3 has a 4096-bit RSA key and sha256 signature algorithm support added to it.
```

File: product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx [hunk 6]
```diff
@@ -5,56 +5,73 @@ navTitle: "Prerequisites"
...
+  1. Set up the repository
+
+     Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
+
+     -  To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+
+     -  To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+   
+   !!! Note
+       Setup the PostgreSQL community repository is required only if you are using PostgreSQL as the backend database for PEM server.
```
Comment (sonotley) [inline product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx:17]:
```suggestion
       The PostgreSQL community repository is required only if you are using PostgreSQL as the backend database for PEM server.
```

**Review by sonotley — APPROVED** [review]
Looks good, I have approved but also suggested two minor grammatical improvements.

**Comment by nidhibhammar** [comment]
@drothery-edb, I think we are good to merge this PR.

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**